tests/foreman/cli/test_role.py

```diff
@@ -43,19 +43,55 @@ def test_positive_create_with_name(self):
                 role = make_role({'name': name})
                 self.assertEqual(role['name'], name)
 
-    @skip_if_bug_open('bugzilla', 1138559)
-    @stubbed
-    def test_positive_create_with_permission(self):
+    @tier1
+    def test_positive_create_with_filter(self):
+        """Create new role with a filter
+
+        @id: 6c99ee25-4e58-496c-af42-f8ad2da6cf07
+
+        @assert: Role is created and correct filter is assigned
+        """
+        role = make_role({'name': gen_string('alpha')})
+        # Pick permissions by its resource type
+        resource_type = 'Organization'
```

Comment:
Whether put it like a const on a module level or use it as a string directly in the call
We do not use variables for one time usage